tests/regression_tests/unstructured_mesh/inputs_true1.dat

```diff
@@ -57,35 +57,34 @@
         <parameters>0.0 1.0</parameters>
       </phi>
     </space>
-    <angle reference_uvw="-1.0 0.0 0.0" type="monodirectional" />
     <energy type="discrete">
       <parameters>15000000.0 1.0</parameters>
     </energy>
   </source>
 </settings>
 <?xml version='1.0' encoding='utf-8'?>
 <tallies>
-  <mesh id="3">
+  <mesh id="1">
     <dimension>10 10 10</dimension>
     <lower_left>-10.0 -10.0 -10.0</lower_left>
     <upper_right>10.0 10.0 10.0</upper_right>
   </mesh>
-  <mesh id="4" type="unstructured">
-    <filename>test_mesh_tets.h5m</filename>
+  <mesh id="2" library="libmesh" type="unstructured">
+    <filename>test_mesh_tets.exo</filename>
```

Comment:
In `LibMesh::write(...)`, the message suggests you're writing a file with `.e` extension, but here you are supplying a file with `.exo` extension. In my experience these two extensions are interchangeable (can just rename `cyl.exo` to `cyl.e` and things work fine), but I think it'd be nice for consistency purposes to just stick with one or the other from a documentation POV.

Same comment applies for several other files in this PR.

Reply:
Good point. Best to be consistent about that. I'll opt for `.e` and update for consistency.